src/main/java/fi/dy/masa/litematica/schematic/conversion/SchematicConversionFixers.java

```diff
@@ -57,7 +57,7 @@ public class SchematicConversionFixers
     public static final IStateFixer FIXER_BANNER = (reader, state, pos) -> {
         NbtCompound tag = reader.getBlockEntityData(pos);
 
-        if (tag != null)
+        if (tag != null && tag.contains("Base"))
```

Comment:
These tags need to be checked for, because this Whole Class does break NBT data from working properly in 1.20.X at Litematic load.  Simply checking for the tags fixes the issue.